templates/financial_info/edit.html.twig

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}New Booking{% endblock %}
+{% block header %}
+{% endblock header %}
```

Comment:
Why completing a block only with spaces and new lines ?

Reply:
It was for the future navbar, but it's removed now.